xml/src/test/java/org/ehcache/xml/FromTemplateCacheConfigurationBuilderDefaultTest.java

```diff
@@ -38,7 +38,7 @@ public void setUp() throws Exception {
   }
 
   @Test
-  public void testDefaultExpiry() throws Exception {
-    assertThat(minimalTemplateBuilder.hasDefaultExpiry(), is(true));
+  public void testNoConfiguredExpiry() throws Exception {
+    assertThat(minimalTemplateBuilder.hasConfiguredExpiry(), is(false));
```

Comment:
What's the story behind this test change? Is that related to the inverted condition above?